src/astro/sql/table.py

```diff
@@ -19,8 +21,8 @@ class Metadata:
     """
 
     # This property is used by several databases, including: Postgres, Snowflake and BigQuery ("namespace")
-    schema: Union[str, None] = None
-    database: Union[str, None] = None
+    schema: str | None = None
```

Comment:
We should use `Optional[str]` instead

Reply:
I will follow it up with a PR

Reply:
This was changed by `Black` as part of our pre-commits, as explained here:
https://github.com/astronomer/astro-sdk/pull/384#discussion_r878956136